app/src/androidTest/kotlin/at/bitfire/davdroid/servicedetection/CollectionListRefresherTest.kt

```diff
@@ -97,7 +98,10 @@ class CollectionListRefresherTest {
         val baseUrl = mockServer.url(PATH_CARDDAV + SUBPATH_PRINCIPAL)
 
         // Query home sets
-        refresherFactory.create(service, client.okHttpClient).discoverHomesets(baseUrl)
+        refresherFactory.create(service, client.okHttpClient).apply {
+            discoverHomesets(baseUrl)
+            refreshHomesetsAndTheirCollections()
```

Comment:
`discoverHomesets()` should already insert the personal flag.

It should call the same methods as `refreshHomesetsAndTheirCollections`.